WindowsDevicePortalWrapper/WindowsDevicePortalWrapper/DefaultDevicePortalConnection.cs

```diff
@@ -151,11 +173,19 @@ public void UpdateConnection(bool requiresHttps)
                     // We take the first, non-169.x.x.x address we find that is not 0.0.0.0.
                     if ((addressInfo.Address != "0.0.0.0") && !addressInfo.Address.StartsWith("169."))
                     {
+                        string port = "";
+                        string[] addressParts = this.Connection.Authority.Split(':');
```

Comment:
This looks like it will pull over the port (10080) when used over USB, despite it likely no longer being relevant. Is that the case?